macros/edr/materializations/tests/handle_failed_test_result_count.sql

```diff
@@ -0,0 +1,63 @@
+{% macro handle_failed_test_result_count(flattened_test) %}
```

Comment:
I think that it'd be better to name this something like `cache_failed_test_result_count` in order to imply what it actually does.

Reply:
but it runs the query as well, so i dont think `cache` is the correct word here.
I agree that it isnt the best name, but I followed the convention of `handle_test_results`,

Reply:
Got it, this makes sense to me.
What do you think about changing this macro to only return the fail count and rename to something very simplistic such as `get_test_fail_count` and then cache its value [here](https://github.com/elementary-data/dbt-data-reliability/blob/fc6e2caeff13e9d363ae6394217ab8cbbce4852a/macros/edr/materializations/tests/test.sql#L96)?

Reply:
That seems right, will do